lib/redux/inventory/actions.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:convert';
+
+import 'package:redux/redux.dart';
+import 'package:redux_thunk/redux_thunk.dart';
+import 'package:http/http.dart' as http;
+import 'package:scibble/models/product.dart';
+
+import '../store.dart';
+
+class SetInventory {
+  final List<Product> payload;
+  SetInventory(this.payload);
+}
+
+ThunkAction<AppState> getInventory = (Store<AppState> store) async {
```

Comment:
Might be an idea to abstract
```
  final token = store.state.auth.token;
  var response = await http.get(
    'https://online.ntnu.no/api/v1/URL
    headers: {'Authorization': 'Bearer ${token.accessToken}'},
  );
```

Into its own generic function, as it is being used in two places - and will definitely be used in more.